modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -81,7 +81,7 @@
 	alt_titles = list("Geneticist", "Mutation Researcher")
 
 /datum/job/head_of_personnel
-	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor")
+	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor", "Enterprise Resource Planner", "Chief Resources Officer")
```

Comment:
Let's not add 'ERP HOP' and pretend we did.

Reply:
I didn't want to call it Human, because we have a bunch of species.

Reply:
It's a joke name, sir.

Reply:
enterprise resource planner is also better as a QM name, but also lets not add ERP HOP

Reply:
erotic role play head of personnel